scripts/run-framework/agief_experiment/experiment.py

```diff
@@ -113,10 +113,10 @@ def log_results_config(self, compute_node):
                     param_path = config_exp['value'][reporting_path_key]
                     report = dpath.util.get(config, 'value.' + param_path, '.')
                 else:
-                    print("WARNING: No reporting entity config path found in experiment config.")
+                    logging.warning("No reporting entity config path found in experiment config.")
             except KeyError:
-                print("KeyError Exception")
-                print("WARNING: trying to access path '" + param_path + "' at config.value, but it DOES NOT exist!")
+                logging.error("KeyError Exception: trying to access path '" + 
```

Comment:
This is a KeyError exception, but from the python script's perspective, it isn't serious enough to be an Error.
Could you please change to log.warning rather than log.error

Error should be something that has gone seriously wrong and serious enough to not continue.
Warning, it can continue and still be successful. This KeyError results in incomplete logging of the results, but they're still available, just not in the console. 

In light of this definition, and to be consistent, there may be other places that we should change from Error to Warning.